app/queues/common/expiration-request.server.ts

```diff
@@ -37,23 +42,41 @@ const expirationRequestWorker = new Worker(
       logger.info('process DNS record expiration');
       let dnsRecords = await getExpiredDnsRecords();
       await Promise.all(
-        dnsRecords.map(async ({ id, user }) =>
+        dnsRecords.map(async ({ id, subdomain, user }) =>
           Promise.all([
-            // delete records from Route53 and DB
+            // delete DNS records from DB
             deleteDnsRecordById(id),
-            // add notification jobs (assuming deletion went successfully)
+            // add notification jobs
             addNotification({
               emailAddress: user.email,
-              subject: 'DNS record expiration subject',
-              message: 'DNS record expiration message',
+              subject: 'My.Custom.Domain DNS record expired',
+              message: `${user.displayName}, your DNS record with subdomain: ${subdomain} has expired.`,
             }),
           ])
         )
       );
     } catch (err) {
       throw new UnrecoverableError(`Unable to process DNS record expiration: ${err}`);
     }
-    logger.info('TODO: process certificate expiration');
+    try {
+      logger.info('process certificate expiration');
+      let certificates = await getExpiredCertificates();
+      certificates.map(async ({ id, username, domain, user }) => {
+        let mostRecentCertificate = await getCertificateByUsername(username);
+        // add notification jobs
```

Comment:
Can you add a comment to explain this extra check?  I think what I'm reading here is that you'll clean-up old certificates without notifying, but if this is the current certificate, you'll send a notification.  Is that right?  If so, please say that.